public/Lab-Files/Lesson02/Challenges/blog-post.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+</head>
+<body>
+    <h1 style="color:red">El dia de los Blogs.</h1>
+    
+    <p>This blog will be dedicated to all things paranormal, from the well known<br>
+        "Mothman" or "La Llorona" to tales i've have heard from friends to<br>
+        stories I heard from my grandmother as a child, and of course my own<br>
+        personal experiences. I find it curious how this subject frigthens people<br>
+        but also intrigues them, and you can include me in that group. I remember<br>
+        my brother and I as children begging my <em>abuela</em> to tell us <strong> ghost stories</strong><br>
+        but then at night having to sleep with the lights on, because as soon as the<br>
+        lights went off, all those stories came to life or death, I guess depending on<br>
+        how you looked at it. But every time we saw my grandmother, there we would go<br>
+        again pestering her until she broke down and told us a story<br>
+    </p>
+    
+    <p>So, this is the first of many posts to come. I hope it will entertain some,<br>
+        scare others, and maybe remind some of things they may have heard or seen<br>
+        but had forgotten or maybe wanted to forget. So here we go, my abuela<br>
+        told us a story about............... 
+        </p>
+
+</body>
+</html>
```

Comment:
Great job on this assignments, there were no errors at all. I can tell you really took the time to implement a lot of the concepts you have been learning.